code/modules/mob/living/carbon/alien/humanoid/emote.dm

```diff
@@ -60,6 +60,26 @@
 				message = text("<span class='name'>[src]</span> signs[].", (text2num(param) ? text(" the number []", text2num(param)) : null))
 				m_type = 1
 
+		if ("special")	//visual emotes with their own special sprites - DO NOT BE BLIND LIKE A CERTAIN SOMEONE WHO SHOULD HAVE ASKED WHAT EACH SPRITE WAS
+			if (sprite_changed_for_emote)
+				src << "You are currently brimming with emotion!"

```

Comment:
Sometimes switches drop and sometimes they don't. That's BYOND for you. Here you might want a more descriptive message like "You're already" instead of "You are" and use spans (as mentioned before).